src/main/java/mate/academy/hibernate/relations/model/Actor.java

```diff
@@ -1,8 +1,20 @@
 package mate.academy.hibernate.relations.model;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "actors")
 public class Actor implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String name;
+    @OneToOne
```

Comment:
I think here we should use `@ManyToOne` instead because an actor can belong only to one single country and the country could have many actors. What do you think ? (but I am not 100% sure, let the mentor decide)